src/main/scala/chisel3/Driver.scala

```diff
@@ -241,10 +241,21 @@ object Driver extends BackendCompilationUtilities {
     val annosx = try {
       phases.foldLeft(annos)( (a, p) => p.transform(a) )
     } catch {
-      /* ChiselStage and FirrtlStage can throw StageError. Since Driver is not a StageMain, it cannot catch these. While
-       * Driver is deprecated and removed in 3.2.1+, the Driver catches all errors.
+      /* Catch all errors and show a fake stack trace. If the stack trace was minified elsewhere, e.g., in the Elaborate
+       * phase, then it will show up minified here. The Driver cannot fail for any reason. Instead, a
+       * ChiselExecutionFailure result will show up when trying to view the resulting annotation sequence.
        */
-      case e: StageError => annos
+      case a: Throwable =>
```

Comment:
Rather than `Throwable`, I think you should use `NonFatal`: https://www.scala-lang.org/api/2.12.11/scala/util/control/NonFatal$.html

This ensures we don't accidentally catch some exceptions where trying to continue executing would be problematic.